server/jetstream_api.go

```diff
@@ -163,167 +163,213 @@ type ApiError struct {
 	Description string `json:"description,omitempty"`
 }
 
+// ApiResponse is a standard response from the JetStream JSON API
+type ApiResponse struct {
+	Type  string    `json:"type"`
+	Error *ApiError `json:"error,omitempty"`
+}
+
+// ApiPagedResponse includes variables used to create paged responses from the JSON API
+type ApiPagedResponse struct {
+	Total  int `json:"total"`
+	Offset int `json:"offset"`
+	Limit  int `json:"limit"`
+}
+
+// ApiPagedRequest includes parameters allowing specific pages to be requests from APIs responding with ApiPagedResponse
+type ApiPagedRequest struct {
+	Offset int `json:"offset"`
+}
+
 // JSApiAccountInfoResponse reports back information on jetstream for this account.
 type JSApiAccountInfoResponse struct {
-	Error *ApiError `json:"error,omitempty"`
+	ApiResponse
 	*JetStreamAccountStats
 }
 
+const JSApiAccountInfoResponseType = "io.nats.jetstream.api.v1.account_info_response"
```

Comment:
I will follow your lead here, but was wondering if shorten jetstream and make request and response a token?

e.g. `io.nats.js.api.v1.response.account_info`

Reply:
How do we determine where the version goes? Does this mean any change would be v2 for the whole thing? So you can't upgrade a single api request or response?

Reply:
I'd be reluctant to change to `js` now just because its a crazy amount of work, we discussed and agreed on what we have :)

You dont need to version bump the whole API, no